calico-policies/public-network-isolation/au-syd/allow-public-services.yaml

```diff
@@ -64,24 +64,7 @@ spec:
       - 15443
     protocol: TCP
     source: {}
-  ingress:
```

Comment:
This section can stay in the policies that apply to `ibm.role == 'worker_public'`, since the selector for the policy is the HostEndpoint with the `ibm.role == 'worker_public'`.  In this case the HostEndpoint is what denies the traffic if there is no `Allow` rule for it (unlike the policies that apply to pods, that use the `projectcalico.org/orchestrator == 'k8s'` selector).

Reply:
ahh nice, okay fixing